build/Palaso.proj

```diff
@@ -87,11 +87,7 @@
 			WorkingDirectory="$(RootDir)/output/$(Configuration)"
 			Force32Bit="$(useNUnit-x86)"
 			Verbose="true"
-			OutputXmlFile="$(TestOutputXmlFile)"
```

Comment:
If you're getting rid of TestOutputXmlFile, you can get rid of the lines above where it gets set.

Reply:
That's true. Unfortunately it doesn't work the way I had hoped it would - TC still doesn't see the individual tests. Need to do some more investigations when I find some time (and add this line back in).

Reply:
The thing to get it working was to install the two NUnit extension nuget packages.